tests/main/apparmor-prompting-client-integration-tests/task.yaml

```diff
@@ -0,0 +1,78 @@
+summary: Run the prompting-client integration tests against snapd
+
+details: |
+    This test checks that the integration tests for the prompting-client pass
+    when run against snapd.
+
+systems:
+  - ubuntu-2*
+
+prepare: |
+    # Prerequisite for having a prompt handler service
+    snap set system experimental.user-daemons=true
+    "$TESTSTOOLS"/snaps-state install-local test-snapd-prompt-handler
+    snap connect test-snapd-prompt-handler:snap-interfaces-requests-control
+
+    snap install lxd
+    lxd init --auto
+    snap install snapcraft --classic
+
+    git clone https://github.com/canonical/prompting-client /home/ubuntu/prompting-client
```

Comment:
It might be a good idea to use relative paths when building If this test is ever extended to more distros.